all_repos/push/bitbucket_server_pull_request_bearer_auth.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from typing import NamedTuple
+
+from all_repos.push.bitbucket_server_pull_request import _push
```

Comment:
you shouldn't import underscored names.

also this is over-DRY, don't be afraid of copying a little bit of code vs. making spaghetti

Reply:
I will refactor common code into a separate package. I don't think it is too much DRY. In fact, the only difference is auth scheme.

Reply:
@asottile Can you please take a look now?